common/adminBlock/EntryAddFactoidAddress.go

```diff
@@ -32,6 +34,7 @@ func (e *AddFactoidAddress) Init() {
 	}
 }
 
+// IsSameAs returns true iff the input is identital to this object
 func (a *AddFactoidAddress) IsSameAs(b *AddFactoidAddress) bool {
```

Comment:
(a *AddFactoidAddress) should be (e *AddFactoidAddress)?

Reply:
done